src/module/effect/SR5ActiveEffect.ts

```diff
@@ -398,11 +397,14 @@ export class SR5ActiveEffect extends ActiveEffect {
         /**
          * label -> name
          * @deprecated since v11
-         */
-        // @ts-expect-error TODO: foundry-vtt-types v10
-        this._addDataFieldMigration(data, "label", "name", d => d.label || "Unnamed Effect");
+        */
+       // @ts-expect-error TODO: foundry-vtt-types v10
+       this._addDataFieldMigration(data, "label", "name", d => d.label || "Unnamed Effect");
```

Comment:
can I just do: `data.name = data.label || "Unnamed Effect` on `Version0_18_0`? or is it intrinsically different?